encryptor/queryDataEncryptor.go

```diff
@@ -95,8 +115,8 @@ func (encryptor *QueryDataEncryptor) encryptInsertQuery(ctx context.Context, ins
 			for _, valTuple := range rows {
 				// collect values per column
 				for j, value := range valTuple {
-					columnName := columnsName[j]
-					if changedValue, err := encryptor.encryptExpression(ctx, value, schema, columnName); err != nil {
+					columnName := columnsName[j%len(columnsName)]
```

Comment:
Why is there a modulo operation?

Reply:
In the vscode plugin that I use there is a support of additional diagnostics about gc-escapes and small optimizations that guarantee to apply. And when I change this to just `j`, this what the vscode shows:
![image](https://user-images.githubusercontent.com/43041209/160286685-2797ac18-9051-4417-913c-85f474b00c82.png)

That means the compiler was able to deduce that `j` is in bounds of the `columnsName` slice.

Reply:
we use module because there should be not more than declared columns count:
`insert into table1(col1, col2, col3) values (val1, val2, val3), (val4, val5, val6);`
There is we have 3 columns and 3 tuples. Every tuple can have 3 values. If more than it's invalid behavior and we can skip it and pass as is to database. Because database will deny such query. But in valid case we are safe and map value to correct column.

Reply:
Ok, this is reasonable. But in my opinion, it would be better to just check that if `j > len(columnNames)` then just skip this iteration with `continue`. At least we will not spend time encrypting extra values.